internal/provider/route_model.go

```diff
@@ -23,6 +25,7 @@ type RouteModel struct {
 	HostRewriteHeader                         types.String         `tfsdk:"host_rewrite_header"`
 	ID                                        types.String         `tfsdk:"id"`
 	IdleTimeout                               timetypes.GoDuration `tfsdk:"idle_timeout"`
+	IDPAccessTokenAllowedAudiences            types.List           `tfsdk:"idp_access_token_allowed_audiences"`
```

Comment:
```suggestion
	IDPAccessTokenAllowedAudiences            types.Set           `tfsdk:"idp_access_token_allowed_audiences"`
```

as order is insignificant